ui/desktop/electron-app/config/forge.config.js

```diff
@@ -137,21 +137,5 @@ module.exports = {
         });
       });
     },
-    packageAfterPrune: async (_, buildPath) => {
```

Comment:
since we are pinning node-gyp v10, we don't need this workaround anymore, I tested it, but lmk if there's anything I missed